agent/config/config_unix.go

```diff
@@ -28,6 +28,7 @@ func DefaultConfig() Config {
 		ReservedPorts:               []uint16{SSHPort, DockerReservedPort, DockerReservedSSLPort, AgentIntrospectionPort, AgentCredentialsPort},
 		ReservedPortsUDP:            []uint16{},
 		DataDir:                     "/data/",
+		DataDirOnHost:               "/var/lib/ecs",
```

Comment:
I've added a configuration for this temporarily but I have realized I need to do some changes to make configuration of this feature make more sense from a user perspective.